bots/mailman/bot.js

```diff
@@ -1,44 +1,91 @@
+function locationsSuggestions (params) {
+    var result = {title: "Send location"};
+    var seqArg = params.seqArg ? params.seqArg : "";
+
+    if (seqArg == "Dropped pin")
+    {
+        result.showBack = true;
+        result.height = "max";
+        result.markup = ["view", {},
+                            ['dropped-pin']];
+    }
+    else if (seqArg != "")
+    {
+        result.showBack = true;
+        result.markup = ['scroll-view', {keyboardShouldPersistTaps: "always"},
+                            ['view', {},
+                                ['places-search']]];
+    }
+    else
+    {
+        result.markup = ['scroll-view', {keyboardShouldPersistTaps: "always"},
+                            ['view', {},
+                                ['current-location-map'],
+                                ['current-location'],
+                                ['separator'],
+                                ['places-nearby']]];
```

Comment:
i haven't seen conventions or rules, and i decided we can use both, for me it's more convenient to use strings, so i decided to use strings, but i think i forget to create functions for new components, i'll create functions